state-sync/diem-data-client/src/diemnet/metrics.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use diem_crypto::_once_cell::sync::Lazy;
+use diem_metrics::{
+    register_histogram_vec, register_int_counter_vec, HistogramTimer, HistogramVec, IntCounterVec,
+};
+
+pub const TOTAL: &str = "TOTAL";
+
+/// Counter for tracking sent requests
+pub static SENT_REQUESTS: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "diem_data_client_sent_requests",
+        "Counters related to sent requests",
+        &["request_types"]
+    )
+    .unwrap()
+});
+
+/// Counter for tracking success responses
+pub static SUCCESS_RESPONSES: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
I'm thinking of keep tracking of success responses/error responses per request type and a total of successes/errors. If you'd like to simplify by keeping track less things, please let me know.

Reply:
Yeah, I think this makes sense 😃